scripts/master.py

```diff
@@ -529,18 +529,6 @@
         },
         KERAS_CORE_MASTER,
         EXAMPLES_MASTER,  # The examples section master will be mostly autogenerated.
-        {
-            "path": "why_keras",
-            "title": "Why choose Keras?",
-        },
-        {
-            "path": "governance",
-            "title": "Community & governance",
-        },
-        {
-            "path": "contributing",
-            "title": "Contributing to Keras",
-        },
         {
```

Comment:
I might find things more scannable if we grouped things better. And potentially added a caption for all our libraries (similar to what we have for KerasCore?).

About
Getting Started
Guides
Examples
API Reference
Keras Core: Keras for TensorFlow, JAX, and PyTorch
Keras Tuner: Hyperparameter Tuning
Keras CV: Computer Vision Workflows
Keras NLP: Natural Language Workflows

Then we are keeping "like next to like." E.g. guides next to examples. Ecosystem libraries next to each other. @fchollet what do you think?

Reply:
Oh also, let's rename the `Keras API Reference` section to just `API Reference`. It's implied that everything is Keras, and we also list the domain package APIs there.

Reply:
Thanks for reviewing, I made the changes as you suggested, please review the below screenshot before I make commit.

![image](https://github.com/keras-team/keras-io/assets/73069040/15e9bbe6-5d47-4830-b07c-4075d98e0014)